code/game/machinery/cloning.dm

```diff
@@ -216,9 +216,9 @@
 		else
 			locked = 0
 			user << "System unlocked."
-	else if(istype(I, /obj/item/weapon/reagent_containers/food/snacks/meat))
+	else if(I.type in BIOMASS_TYPES)
```

Comment:
Can you use `istype` here to include biomass subtypes too?

Reply:
fixed